.travis.yml

```diff
@@ -38,10 +38,6 @@ addons:
 
 env:
   - FCFLAGS_ADD='' DISTCHECK_CONFIGURE_FLAGS=''
-  - FCFLAGS_ADD='' DISTCHECK_CONFIGURE_FLAGS='--without-openmp'
-  - FCFLAGS_ADD='' DISTCHECK_CONFIGURE_FLAGS='--enable-mixed-mode'
-  - FCFLAGS_ADD='-fdefault-real-8 -fdefault-double-8 -fcray-pointer -ffree-line-length-none' DISTCHECK_CONFIGURE_FLAGS='--disable-setting-flags'
-  - FCFLAGS_ADD='' DISTCHECK_CONFIGURE_FLAGS='--enable-mpp-io'
```

Comment:
Why were these tests removed?  They should be added back.

Reply:
`master` `.travis.yml` is now restored.
This uses `autotools` and does not test `cmake` build.

Reply:
@thomas-robinson I have restored `.travis.yml` to its `master` form, but that will break the build test.
I don't see the point in `autotools` generating `cmake` files, then `cmake` generating `Makefile` for build.
Adding just those flags means nothing to `cmake`.